sqlconnect/internal/postgres/testdata/column-mapping-test-seed.sql

```diff
@@ -30,8 +33,8 @@ CREATE TABLE "{{.schema}}"."column_mappings_test" (
 );
 
 INSERT INTO "{{.schema}}"."column_mappings_test"
-    (_order, _int, _int2, _int4, _int8, _integer, _smallint, _bigint, _real, _float, _float4, _float8, _numeric, _double, _text, _varchar, _charvar, _nchar, _bpchar, _character, _timestamptz, _timestampntz, _timestampwtz, _timestamp, _boolean, _bool, _json, _jsonb)
+    (_order, _int, _int2, _int4, _int8, _integer, _smallint, _bigint, _real, _float, _float4, _float8, _numeric, _double, _text, _varchar, _charvar, _nchar, _bpchar, _character, _date, _time, _timetz, _timestamptz, _timestampntz, _timestampwtz, _timestamp, _boolean, _bool, _json, _jsonb)
 VALUES
-    (1, 1,    1,    1,    1,    1,    1,    1,    1.1,  1.1,  1.1,  1.1,  1.1,  1.1,  'abc', 'abc', 'abc', 'abc', 'abc', 'abc', '2004-10-19 10:23:54+02', '2004-10-19 10:23:54', '2004-10-19 10:23:54+02', '2004-10-19 10:23:54+02', true,  true,  '{"a": 1}', '{"a": 1}'),
-    (2, 0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    0,    '',    '',    '',    '',    '',    '',    '2004-10-19 10:23:54+02', '2004-10-19 10:23:54', '2004-10-19 10:23:54+02', '2004-10-19 10:23:54+02', false, false, '{}',       '{}'      ), 
-    (3, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL,  NULL,  NULL,  NULL,  NULL,  NULL,  NULL,                     NULL,                  NULL,                     NULL,                     NULL,  NULL,  NULL,       NULL      );
\ No newline at end of file
+    (1, 1,    1,    1,    1,    1,    1,    1,    1.1,  1.1,  1.1,  1.1,  1.1,  1.1,  'abc', 'abc', 'abc', 'abc', 'abc', 'abc', '2004-10-19', '10:23:54', '10:23:54+02', '2004-10-19 10:23:54+02', '2004-10-19 10:23:54', '2004-10-19 10:23:54+02', '2004-10-19 10:23:54+02', true,  true,  '{"a": 1}', '{"a": 1}'),
```

Comment:
value for time and timetz looks off. 🤔

Reply:
Please elaborate, I don't understand what is the concern:
- `_time '10:23:54'` gets mapped to `"0000-01-01T10:23:54Z"`
- `_timetz '10:23:54+02'` gets mapped to `"0000-01-01T10:23:54+02:00"`

One is without a timezone and the other one is with a timezone, they are not equal